edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss

```diff
@@ -1,7 +1,358 @@
+@import 'course-experience';
+
+
+//registration
+.view-register .form-actions button[type="submit"]:hover {
+  background-color:#fff;
+  border: 1px solid #007FA3;
+}
+
+
+.register-form {
+  max-width: 600px;
+  margin: auto;
+}
+//generals
+
+body, html, button, textarea {
+  color:$primary-font-color ;
+}
+
+html {
+  background: $inverse !important;
+}
+
+input::not(input[type="button"]) {
+  color:$primary-font-color !important;
+}
+
+select, option, optgroup {
+  color: $primary-font-color !important;
+  background-color: $inverse !important;
+  -webkit-appearance: menulist !important;
+  -moz-appearance: menulist !important;
+  -ms-appearance: menulist !important;
+  -o-appearance: menulist !important;
+}
+
+textarea {
+  color: $primary-font-color !important;
+  background-color: $inverse !important;
+}
+
+a, a:visited:not(.btn) {
+  color: $primary ;
+}
+
+input[type="text"],
+input[type="email"],
+input[type="password"] {
+  color: $primary-font-color;
+}
+
+
+.light-button:hover:not(:disabled),
+div.history-controls input[type="submit"]:hover:not(:disabled),
+a.light-button:hover:not(:disabled),
+div.info-wrapper section.handouts ol li div.hitarea:hover+h4.light-button:not(:disabled),
+div.info-wrapper section.handouts ol li div.hitarea:focus+h4.light-button:not(:disabled),
+.gray-button:hover:not(:disabled),
+input[type="reset"]:hover:not(:disabled),
+input[type="submit"]:hover:not(:disabled),
+input[type="button"]:hover:not(:disabled),
+button:hover:not(:disabled),
+.button:hover:not(:disabled),
+.xmodule_display.xmodule_LTIModule div.lti.wrapper-lti-link
+.lti-link .link_lti_new_window:hover:not(:disabled),
+input[type="submit"].btn-blue:hover:not(:disabled),
+input.btn-blue:hover:not(:disabled),
+button.btn-blue:hover:not(:disabled) {
+  background: $primary;
+  background-image: none;
+}
+
+
+
+.field-label{
+  color: $primary-font-color;
+}
+
+.field-input{
+  background-color: $inverse;
+}
+
+.btn {
+  border-color: $inverse;
+  background: $primary ;
+  color:$inverse ;
+}
+
+.btn:hover, .is-hovered.btn, .btn:focus, .is-focused.btn {
+  border-color: $inverse;
+  background: $primary ;
+}
+
+div.page-banner div.alert.alert-warning {
+  background-color: $gray-l5;
+  border-color: $gray-l5;
+}
+
+div.prompt.warning.has-actions
+nav.nav-actions li.nav-item button.action-primary {
+  background-color: $primary;
+  border: 1px solid $primary;
+  border-color: $primary;
+}
+
+.wrapper-prompt div.prompt.warning div.copy {
+  background: $gray-l5;
+  border-top-color: $primary;
+}
+
+.contact-us-wrapper .btn-primary,
+.contact-us-wrapper .btn-brand {
+  border-color: $primary;
+}
+
+.contact-us-wrapper .btn-primary:hover,
+.contact-us-wrapper .btn-brand:hover,
+.contact-us-wrapper .btn-primary:focus,
+.contact-us-wrapper .btn-brand:focus {
+    background-image: none;
+    background-color: $primary;
+    border-color: $primary;
+}
+
+@media (min-width: 768px) {
+  .content-wrapper {
+     padding: 0 0 10px;
+  }
+}
```

Comment:
Can you arrange this in a media query section, at the bottom of the file? including the other media query definitions... @luismorenolopera

Reply:
done